cfgov/wellbeing/urls.py

```diff
@@ -10,17 +11,60 @@
 
 
 urlpatterns = [
-    re_path(
-        r'^$(?i)',
-        TemplateView.as_view(template_name='wellbeing/home.html'),
-        name='fwb_home'
-    ),
-    re_path(
-        r'^results/$(?i)', ResultsView.as_view(), name='fwb_results'
-    ),
-    re_path(
-        r'^about/$(?i)',
-        TemplateView.as_view(template_name='wellbeing/about.html'),
-        name='fwb_about'
+    url(
+        r'^$',
+        TranslatedTemplateView.as_view(
+            template_name='wellbeing/home.html'
+        ),
+        name='fwb_home_en'
+    ),
+    url(
+        r'^results/$',
+        ResultsView.as_view(),
+        name='fwb_results_en'
+    ),
+    url(
+        r'^about/$',
+        TranslatedTemplateView.as_view(
+            template_name='wellbeing/about.html'
+        ),
+        name='fwb_about_en'
+    ),
+    url(
+        r'^error/$',
+        TranslatedTemplateView.as_view(
+            template_name='wellbeing/error.html'
+        ),
+        name='fwb_error_en'
+    ),
+
+    url(
+        r'^es/$',
+        TranslatedTemplateView.as_view(
+            template_name='wellbeing/home.html',
+            language='es'
+        ),
+        name='fwb_home_es'
+    ),
+    url(
+        r'^results/es/$',
+        ResultsView.as_view(language='es'),
+        name='fwb_results_es'
+    ),
+    url(
+        r'^about/es/$',
+        TranslatedTemplateView.as_view(
+            template_name='wellbeing/about.html',
+            language='es'
+        ),
+        name='fwb_about_es'
+    ),
+    url(
+        r'^error/es/$',
+        TranslatedTemplateView.as_view(
+            template_name='wellbeing/error.html',
+            language='es'
+        ),
+        name='fwb_error_es'
```

Comment:
I think the `re_path` removal above was a casualty of merging. We should replace the urls below with re_path, since that's what we standardized on going forward for Django 2.

```suggestion
    re_path(
        r'^$',
        TranslatedTemplateView.as_view(
            template_name='wellbeing/home.html'
        ),
        name='fwb_home_en'
    ),
    re_path(
        r'^results/$',
        ResultsView.as_view(),
        name='fwb_results_en'
    ),
    re_path(
        r'^about/$',
        TranslatedTemplateView.as_view(
            template_name='wellbeing/about.html'
        ),
        name='fwb_about_en'
    ),
    re_path(
        r'^error/$',
        TranslatedTemplateView.as_view(
            template_name='wellbeing/error.html'
        ),
        name='fwb_error_en'
    ),

    re_path(
        r'^es/$',
        TranslatedTemplateView.as_view(
            template_name='wellbeing/home.html',
            language='es'
        ),
        name='fwb_home_es'
    ),
    re_path(
        r'^results/es/$',
        ResultsView.as_view(language='es'),
        name='fwb_results_es'
    ),
    re_path(
        r'^about/es/$',
        TranslatedTemplateView.as_view(
            template_name='wellbeing/about.html',
            language='es'
        ),
        name='fwb_about_es'
    ),
    re_path(
        r'^error/es/$',
        TranslatedTemplateView.as_view(
            template_name='wellbeing/error.html',
            language='es'
        ),
        name='fwb_error_es'
```